pkg/bindings/images/types.go

```diff
@@ -175,6 +175,10 @@ type SearchOptions struct {
 	SkipTLSVerify *bool `schema:"-"`
 	// ListTags search the available tags of the repository
 	ListTags *bool
+	// Username for authenticating against the registry.
+	Username *string
+	// Password for authenticating against the registry.
+	Password *string
```

Comment:
Shouldn’t these use `schema:"-"` to ensure that the data does not show up in the request body (where we don’t consume it)?

Reply:
As described above, since it authenticates with the `X-Registry-Auth` HTTP header, I think it would be fine to hide it.